terraform/modules/happy-service-ecs/main.tf

```diff
@@ -24,6 +24,111 @@ resource "aws_ecs_service" "service" {
 
 locals {
   task_definition = [
+    {
+      name      = "datadog-agent"
+      essential = true
+      image     = "public.ecr.aws/datadog/agent:latest"
```

Comment:
can we make this configurable with these as the defaults? one object input variable that has the image_host and version? I have had too many cases where I needed to change the image version of something and I had to re-release the module. It's way easier to have it be configurable with sane defaults.